Components/Alert/Alert.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./alertstyles.css" />
+    <title>Alert Component</title>
+  </head>
+  <body>
+    <h1>Alerts</h1>
+    <h2>Simple Alerts</h2>
+    <div class="alerts">
+      <div class="error-alert-background alert">
```

Comment:
Try to keep the class name consistent
```suggestion
      <div class="alert alert-bg-error">
```